lambdas/file_access_patterns/lambda_for_fileList/index.js

```diff
@@ -10,45 +10,50 @@ function response(statusCode, message) {
     body: message,
   };
 }
-function jsonToBase64(json_data){
-    if(json_data === undefined) return json_data;
-    let encoded = Buffer.from(JSON.stringify(json_data), 'ascii').toString('base64');    
-    return encoded;
+function jsonToBase64(json_data) {
+  if (json_data === undefined) return json_data;
+  let encoded = Buffer.from(JSON.stringify(json_data), "ascii").toString(
+    "base64"
+  );
+  return encoded;
 }
-function base64ToJson(bString){
-    if (bString === undefined) return bString
-    let decoded = JSON.parse(Buffer.from(bString, 'base64').toString('ascii'));
-    return decoded;
+function base64ToJson(bString) {
+  if (bString === undefined) return bString;
+  let decoded = JSON.parse(Buffer.from(bString, "base64").toString("ascii"));
+  return decoded;
 }
-  
-exports.listFiles = async (event) =>{
-    try{
-        const reqBody = event.body;
-        const lastEvaluatedKey = base64ToJson(reqBody.LastEvaluatedKey)
-        var params={
-            TableName:"V-Transfer",
-            ScanIndexForward: false,
-            KeyConditionExpression: "#PK= :pk and begins_with(#SK,:sk)",
-            ScanIndexForward: false,
-            ExpressionAttributeNames:{
-                "#PK": "PK",
-                "#SK": "SK"
-            },
-            ExpressionAttributeValues:{
-                ':pk':`USER#${event.path.u_id}`,
-                ':sk':"FILE#"
-            },                        
-            Limit:10,
-        }
-        if(lastEvaluatedKey !== undefined) params.ExclusiveStartKey = lastEvaluatedKey;
-        const file_data = await dynamo.query(params).promise()        
-        return_data={} 
-        return_data.items=file_data.Items       
-        file_data.LastEvaluatedKey!=undefined?return_data.LastEvaluatedKey = jsonToBase64(file_data.LastEvaluatedKey):"";
-        return response(200,return_data);
-    }
-    catch(err){
-        return response(err.statusCode,err.message);
-    }
 
-}
+exports.handler = async (event) => {
+  try {
+    const reqBody = event.body;
+    const lastEvaluatedKey = base64ToJson(reqBody.LastEvaluatedKey);
+    var params = {
+      TableName: "V-Transfer",
+      ScanIndexForward: false,
+      KeyConditionExpression: "#PK= :pk and begins_with(#SK,:sk)",
+      ScanIndexForward: false,
+      ExpressionAttributeNames: {
+        "#PK": "PK",
+        "#SK": "SK",
+      },
+      ExpressionAttributeValues: {
+        ":pk": `USER#${event.path.userId}`,
+        ":sk": "FILE#",
```

Comment:
where is the time-series query support?? like when the user wants to access a file created in the year 2019 for the month of April or all files uploaded on a particular date?